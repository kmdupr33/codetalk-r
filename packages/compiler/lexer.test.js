const lexer = require("./lexer");

describe("lexer", () => {
  it("detects lists", () => {
    expect(
      lexer
        .reset(
          `
    * hello
    * world
    `
        )
        .next()
    ).toMatchSnapshot();
  });
});
