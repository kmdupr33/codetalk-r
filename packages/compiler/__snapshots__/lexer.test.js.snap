// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lexer detects lists 1`] = `
Object {
  "col": 1,
  "line": 1,
  "lineBreaks": 2,
  "offset": 0,
  "text": "
    * hello
",
  "toString": [Function],
  "type": "UnorderedList",
  "value": "
    * hello
",
}
`;
